tests/fixture/mod.rs

```diff
@@ -0,0 +1,128 @@
+use git2;
+use git_rsl::{self, BranchName, RemoteName};
+use tempfile::{self, TempDir};
+use assert_cli::{Assert, Environment};
+
+#[allow(dead_code)]
+pub struct TestFixture {
+    origin: TempDir,
+    local: TempDir,
+    gpg_home: TempDir,
+    pub env: Environment
+}
+
+use std::fs::File;
+use std::io::Write;
+
+fn write_gpg_keygen_script(out_path: &str) {
+    let mut f = File::create(out_path).unwrap();
+    f.write_all("%no-protection\n\
+                 %transient-key\n\
+                 Key-Type: DSA\n\
+                 Key-Length: 1024\n\
+                 Name-Email: test_user@polysync.io\n\
+                 Expire-Date: 0\n\
+                 %commit\n".as_bytes()
+    ).unwrap();
+}
+
+use std::process::Command;
```

Comment:
You should move this `use` statement into the function below, or put it at the top of the file with the rest.